arcade/gui/examples/widget_gallery.py

```diff
@@ -147,9 +147,9 @@ def __init__(self):
 
         example_text = dedent(
             """
```

Comment:
Remove indent

Reply:
Also, your doing a PR into my fork. How does that work? I'm new to open source contributing so I don't know what I'm doing with this lol.

Reply:
@pushfoo how do I merge this into my thing?
When I try it says: "merge: pushfoo-pushfoo_add__all__to_all - not something we can merge"